statsmodels/sandbox/regression/penalized.py

```diff
@@ -59,8 +59,8 @@ class TheilGLS(GLS):
 
     .. math:: q = R \beta + v
```

Comment:
Better to make the docstring raw and leave single escapes than to mix and match escaped-escapes and non-escape sequences (e.g., \\ and \b)

Reply:
Yah, wasn't super-happy with this mixed case thing.